Add "Running on Linux page" to go with new standard Linux installation packages

fghalasz opened the pull request:
Added a "running on Linux" page to document how to use the new standard Linux installation packages.

Also renamed the files in the content/en/running directory to all start with a number so we could control the order of their appearance in the TOC bar on  the left side of the website.  Previously this was just left up to a alpha sort of their file names.

fghalasz committed:
Add Running on Linux page that describes new Linux installation packages in the Medley repo.

masinter commented:
I thought the "weight" parameter in the title declaration was used to sort pages according to their weight. Would it be better to present the options "in order of difficulty" with prerequisites?
I think the "build" pages would come after the "running" pages.

fghalasz committed:
Reverting filenames in running section back to previous names;  adjusted weights of .md files in running section to set my preferred ordering; added a script in the scripts directory to print out the weights of the files/subdirs within a directory

fghalasz committed:
Ooops. Forgot to git add -u to pick up some of the changes in the last commit.  This picks up those changes.

masinter reviewed (approved):
we can review after deployment but TL;DR (source code) so LGTM

fghalasz commented:
You are right - weights is what is used as the primary page ordering mechanism in Hugo/Docsys.  So I reverted the filenames back and adjusted the weights in the running directory.

Here are the current weights:
```
2       online/
10      build/
50      running-on-linux.md
50      running-on-mac.md
50      running-on-win.md
75      running-with-docker.md
```
Problem is that the sort for the duplicate '50's is based on all kinds of non-relevant parameters.

New weights in this PR:

```
10      online/
20      running-on-linux.md
30      running-on-mac.md
40      running-on-win.md
50      running-with-docker.md
60      build/
```

Took the liberty of moving the build subsection to the end of the section.  This seems much more appropriate ordering.

Finally, I added a weights.sh script to the scripts directory.  This will print out the weights (in ascending order) for the
.md files in a directory as well as the weight in the _index.md file in any subdirs.,  Very helpful when adjusting the presentation order in Hugo/Docsy.

Example:
```
frank@oio:~/il/website$ scripts/weights.sh content/en/running
10      online/
20      running-on-linux.md
30      running-on-mac.md
40      running-on-win.md
50      running-with-docker.md
60      build/
```
